### PR TITLE
Clean processes

### PR DESCRIPTION
**Pull request by pelger**
factored out into windows, and *nix runners, removed profiling code into profileRunner for the attention of @davidmarkclements

### CODE REVIEW

File: lib/directoryWatcher.js [hunk 1]
```diff
@@ -20,10 +20,9 @@ var chokidar = require('chokidar')
 module.exports = function () {
 
   var start = function (container, changedCb, cb) {
-    // todo expose follow sym links and depth as options in the config file default to the below
+    // var watcherArgs = {persistent: true, ignoreInitial: true, followSymlinks: false, depth: 4, usePolling: true}
```
Comment (mcollina) [inline lib/directoryWatcher.js]: can you remove this commented line?

Comment (pelger) [inline lib/directoryWatcher.js]: Yep :)

**Comment by pelger** [comment]
if you look in process runner you will see that it handles node as a container type and pulls the node path from the fuge executable...

**Comment by mcollina** [comment]
@pelger can you paste in a link? I don't see it.

I see https://github.com/apparatus/fuge-runner/pull/38/files#diff-a2ac54e027efc9536019e857fd7ec2a1L43 is removed. It's needed for windows to get the node path on windows.

**Comment by pelger** [comment]
It's here: https://github.com/apparatus/fuge-runner/blob/clean_processes/lib/processRunner.js#L36-L48

**Comment by pelger** [comment]
Also windows is now refactored into it's own runner see windowsRunner.js

**Comment by mcollina** [comment]
Sorry my bad :).

**Pull request merged**